deployment/config-static.js

```diff
@@ -1,22 +1,34 @@
 module.exports = {
 	type: 'object',
 	properties: {
-		cleanURLs: {
+		'public': {
+			type: 'string'
+		},
+		'cleanURLs': {
 			type: [
 				'boolean',
 				'array'
```

Comment:
We should take care of this separately, but `array` is a loose type. We should define explicit array types with the allowed value types inside, otherwise I could put an array of _whatever_ here.